services/notification/notifiers/mixins/message/__init__.py

```diff
@@ -190,7 +190,7 @@ def get_middle_layout_section_names(self, settings):
         return [
             section
             for section in sections
-            if section not in ["header", "newheader", "newfooter"]
+            if section not in ["header", "newheader", "newfooter", "newfiles"]
```

Comment:
adding this here because the [new designs](https://www.figma.com/file/9fHZis0dg7pKAQZt8fAGZO/Patch-only-experimentation?type=design&node-id=233-2675&mode=design&t=g0ejs58N2x3GHegy-0) take out the files section from the drop down part